src/index.js

```diff
@@ -1,11 +1,18 @@
 import './style.css';
+
 function component() {
   const element = document.createElement('div');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = 'Hello Webpack!!!!';
-
+  element.innerHTML = 'Hello Webpack!';
+  element.className = 'hello';
   return element;
 }
```

Comment:
I suggest removing this section since it's not part of the project rrequirements.